### DISCUSSION TITLE
Transaction as budget?

### DISCUSSION THREAD

**Discussion by jnaav** [firefly-iii/firefly-iii#5026] [Firefly III]
Is it possible to include a transaction in a budget? What I want to do is this:

Every month, I have my budget divided by home, food, etc. and I want to add one more that would be 'savings' that will consist of transferring money from one account to another. 

But I do not see how to mark this transfer as a type of budget so it does not subtract it from the total. Any way to do this? Thanks

**Comment by JC5** [comment]
No sorry, you can't budget savings. You have to create a piggy bank for that.

**Reply by jnaav** [reply]
Ok, thanks! I will check piggy banks
